src/services/ActivityService.ts

```diff
@@ -8,6 +8,7 @@ import { ActivityUpdateSchema } from "../models/ActivityUpdateSchema";
 import { HTTPResponse } from "../utils/HTTPResponse";
 import { DynamoDBService } from "./DynamoDBService";
 import {ActivityType, HTTPRESPONSE} from "../assets/enums";
+import {format, subDays} from "date-fns";
```

Comment:
Can we use momentjs instead please?
We had issues with this library on test-results service as it was not allowing UTC date calculation.

Reply:
I don't think we need to do that at the moment as this library is only used for time format. If we need UTC calculation then we can think of replacing this library.

Reply:
I was thinking not using it at all as this could encourage other developers to use this library (imported in the file) and if we were to do other date calculation we could have similar issues.
`.format()` from moment should achieve the same result.
We use moment 2.26.0 in test results
Also `subDays` does not seem to be used in this file anyway so we can remove it.